.github/workflows/daily-vuln-scan.yml

```
@@ -308,3 +308,118 @@ jobs:
           path: zap-reports/
           retention-days: 30
+
+  strix-scan:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Strix
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)